### PR TITLE
fix: added contributions treemap loading view

### LINKED ISSUE #2384: Bug: contributions treemap has no loading state showing erroneous data temporarily
The contributions treemap in the list activity page has no loading state which causes it to look broken temporarily.

It basically has this weird state where it says Root with 0 Contributors.

https://github.com/open-sauced/app/assets/833231/c14741f0-b9c3-41ee-9c66-58e13d84aff1

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Now the contributions treemap component has a loading view.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2384
Relates to #2175

## Mobile & Desktop Screenshots/Recordings

**Before**

https://github.com/open-sauced/app/assets/833231/c14741f0-b9c3-41ee-9c66-58e13d84aff1

**After**

https://github.com/open-sauced/app/assets/833231/4c1c2f0b-c19f-481e-85df-5e99dfd983f7

## Steps to QA

1. Ensure your're logged in.
2. Navigate to a list activity page, e.g. https://deploy-preview-2383--oss-insights.netlify.app/lists/8b371419-6219-4448-9de1-f6293a25f98e/activity
3. In the treemap, click on a repository.
4. A loading view loads (could be short) instead of the previous view that said Root, 0 Contributors.

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [x] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by bdougie — APPROVED** [review]
Tested ok mobile and works as expected

**Review by NsdHSO — APPROVED** [review]
LGTM

File: components/molecules/ContributionsTreemap/contributions-treemap.tsx [hunk 1]
```diff
@@ -48,31 +56,35 @@ export const ContributionsTreemap = ({ setRepoId, repoId, data, color, onClick }
...
-                      }: NodeProps<Datum>) => JSX.Element)
-                }
-                colors={color}
-                nodeOpacity={1}
-                borderWidth={0}
-                onClick={onClick}
-                motionConfig={"default"}
-              />
+              {isLoading ? (
+                <div className="react-loading-skeleton" />
```
Comment (brandonroberts) [inline components/molecules/ContributionsTreemap/contributions-treemap.tsx:60]: Should we use our Skeleton Component here?

Comment (nickytonline) [reply components/molecules/ContributionsTreemap/contributions-treemap.tsx:60]: I was going to, but you have to specify a height. I'm going to update the Skeleton loader to not require a height.

I'll follow up with a small PR for that.

Comment (nickytonline) [reply components/molecules/ContributionsTreemap/contributions-treemap.tsx:60]: Actually, I'll add it to this PR, as it is related.

Comment (brandonroberts) [reply components/molecules/ContributionsTreemap/contributions-treemap.tsx:60]: 👍

Comment (nickytonline) [reply components/molecules/ContributionsTreemap/contributions-treemap.tsx]: Done @brandonroberts

**Review by jpmcb — APPROVED** [review]

**Pull request merged**